lms/templates/fields/field_textarea.underscore

```diff
@@ -27,7 +27,7 @@
             <% } %>
             ><%- value %></span>
         <% } else { %>
-            ><a href="#"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></a>
+            ><button type="button"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></button>
```

Comment:
This does not look OK.

![image](https://cloud.githubusercontent.com/assets/484484/17670690/3cc42350-62e1-11e6-8e02-65528a267ca6.png)